src/interpreter.js

```diff
@@ -162,26 +162,26 @@ function keyVal(value) {
 
 function created(utcSeconds) {
   var date = new Date(Math.floor(utcSeconds) * 1000)
-  var dateString =
-    MONTHS[date.getUTCMonth()] + ' ' +
-    padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
-    padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
-    padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
-    padLeft(date.getUTCSeconds().toString(), 2, '0')
-
+  var dateString = date2string(date)
   return `<span class="created">${utcSeconds.toFixed(6)} </span><span class="date">${dateString} </span>`
 }
 
+function date2string(date) {
+    return MONTHS[date.getUTCMonth()] + ' ' +
+      padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
+      padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
+      padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
+      padLeft(date.getUTCSeconds().toString(), 2, '0')
+}
+
 function tsCreated(ts) {
-  var date = new Date(ts)
-  var dateString =
-    MONTHS[date.getUTCMonth()] + ' ' +
-    padLeft(date.getUTCDate().toString(), 2, ' ') + ' ' +
-    padLeft(date.getUTCHours().toString(), 2, '0') + ':' +
-    padLeft(date.getUTCMinutes().toString(), 2, '0') + ':'  +
-    padLeft(date.getUTCSeconds().toString(), 2, '0')
-
-  return `<span class="created">${padSpacesRight((date.getTime() / 1000).toString(), 14)} </span><span class="date">${dateString} </span>`
+  if (typeof ts  == "string") {
```

Comment:
just replace `==` with `===` (it is more performant)

Reply:
done